funsor/interpreter.py

```diff
@@ -23,6 +23,9 @@
 _INTERPRETATION = None  # To be set later in funsor.terms
 _USE_TCO = int(os.environ.get("FUNSOR_USE_TCO", 0))
 
+# TODO remove this, used temporarily for testing
+_GENERIC_SUBS = int(os.environ.get("FUNSOR_GENERIC_SUBS", 0))
```

Comment:
I added this temporary environment variable for testing, but it should be removed either before merging this PR or in a followup PR that removes non-essential `eager_subs` methods.